Makefile

```diff
@@ -268,6 +268,16 @@ BINARYNAME=$(TARNAME)-$(PLATFORM)-$(ARCH)
 BINARYTAR=$(BINARYNAME).tar.gz
 PKG=out/$(TARNAME).pkg
 PACKAGEMAKER ?= /Developer/Applications/Utilities/PackageMaker.app/Contents/MacOS/PackageMaker
+LEDATATAR=$(TARNAME)-locales-le.tar
+BEDATATAR=$(TARNAME)-locales-be.tar
+LEDATATGZ=$(LEDATATAR).gz
+BEDATATGZ=$(BEDATATAR).gz
+#ICU_DATA_IN=$(firstword $(wildcard deps/icu/source/data/in/icudt*l.dat))
+#ICU_DATA_SRC=out/Release/gen
+ICU_DATA_SRC=deps/icu/source/data/in
+ICU_DATA_IN=$(firstword $(wildcard $(ICU_DATA_SRC)/icudt*[bl].dat))
```

Comment:
I think there are a lot of places already where we assume GNU make, fairly sure gyp will use gmake on the bsds and sunos based systems